test/test.ml

```diff
@@ -12,7 +12,20 @@ let debian_12_ocaml_5 =
     os_version = "12";
     ocaml_package = "ocaml-base-compiler";
     ocaml_version = "5.0";
-    opam_version = "2.1.3";
+    opam_version = "2.1.5";
+    lower_bound = false;
+  }
+
+let debian_12_ocaml_4 =
+  { Solver_service_api.Worker.Vars.
+    arch = "x86_64";
+    os = "linux";
+    os_family = "debian";
+    os_distribution = "debian";
+    os_version = "12";
+    ocaml_package = "ocaml-base-compiler";
+    ocaml_version = "4.14.1";
+    opam_version = "2.1.5";
     lower_bound = false;
   }
```

Comment:
Note: a simpler way to do this (used in `test_multiple_roots`) is `{ debian_12_ocaml_5 with ocaml_version = "4.14" }`.